### ISSUE TITLE
my nodemcu bootloop only when it is connected to the computer

### ISSUE THREAD

**Issue by Hoshroin** [spacehuhn/esp8266_deauther#1609]
> Have you searched for existing (open and closed) issues describing the same problem?

No

**Describe the error**
The model is: ESP8266 NodeMCU v3
When I connect the board to a phone adaptor or any other power supplies, everything works smoothly. But only when I connect the board to my computer, it just starts to bootloop. I can't even use the cil function because of this. I just can't understand what has caused it

```
01:43:24.209 -> 
01:43:24.209 ->  ets Jan  8 2013,rst cause:2, boot mode:(3,6)
01:43:24.209 -> 
01:43:24.209 -> load 0x4010f000, len 3584, room 16 
01:43:24.209 -> tail 0
01:43:24.209 -> chksum 0xb0
01:43:24.209 -> csum 0xb0
01:43:24.209 -> v2843a5ac
01:43:24.209 -> ~ld
```

**To Reproduce**
Connect the board to the computer using an usb cable

**Expected behavior**
It starts to bootloop and the same log shows up again and again after about 1 second.

**Environment (please complete the following information):**
 - OS: Windows 10
 - Version: 2.6.1
 - Model: NodeMCU

**Attempts**
I've tried to erase the flash and then compile using the Arduino IDE for a couple of times
Upload speed: 115200
Flash size: 4MB(FS:1MB OTA:~1019kb)
Erase flash: All flash contents
Deauther config: NodeMCU

**Additional context**
Anyway, I haven't tried to connect it to a different computer, maybe that's the problem?

I've tried to use an OLED screen, and it seemed to reboot shortly after it started scanning
(By the way, when i connect three buttons to GPIO14, 13, 12, they also didn't work)
Before flashing the firmware for the first time, it's ok to be connected to my computer, but after doing it using esptool-gui, the trouble just appeared

Please help

**Comment by ryaneditraj** [comment]
have you tried flashing the firmware again ?

**Comment by Hoshroin** [comment]
yes, and did it for a couple of times, but never solves the problem
